demos/guide/Makefile

```diff
@@ -22,6 +22,12 @@ quic-multi-stream: quic-multi-stream.o
 tls-client-non-block: tls-client-non-block.o
 quic-client-non-block: quic-client-non-block.o
 
+chain: chain.pem
+pkey.pem:
+	openssl genpkey -algorithm rsa -out pkey.pem -pkeyopt rsa_keygen_bits:2048
+chain.pem: pkey.pem
+	openssl req -x509 -new -key pkey.pem -days 36500 -subj '/CN=localhost' -out chain.pem
```

Comment:
perhaps it's a nit, but I would stick to established convention and use `serverkey.pem` and `servercert.pem`, or is there a reason why to diverge?